### PR TITLE
Destroying tickets after user unregisters from a conference

### PR DESCRIPTION
**Pull request by rishabhs95**
fixing bug #821

### CODE REVIEW

File: app/models/registration.rb [hunk 1]
```diff
@@ -36,6 +37,13 @@ def week
 
   private
 
+  def destroy_purchased_tickets
+    ticket_purchased = TicketPurchase.where(conference_id: conference_id)
+    unless ticket_purchased.empty?
+      ticket_purchased.destroy_all
```
Comment (hennevogel) [inline app/models/registration.rb]: Erm this would destroy all TicketPurchase for the conference. For all users. For all tickets.

**Comment by hennevogel** [comment]
Hey @rishabhs95 please do not let this discourage you. Re-work your changes (if you need help let us know) and open a new pull request. I'm looking forward to it :-)

File: app/models/registration.rb (hunk 1 above)
Comment (rishabhs95) [inline app/models/registration.rb]: um yes... i'll just correct it... thanks for the advice btw :+1:

**Comment by rishabhs95** [comment]
@hennevogel ... i think these corrections are important... i'll rework the changes :)

**Comment by rishabhs95** [comment]
@hennevogel i've opened a new PR #876 with the corrections, please review the changes.

**Pull request closed without merging**